Source/WebCore/rendering/line/BreakingContext.h

```diff
@@ -178,8 +178,8 @@ class BreakingContext {
 
     LineWidth m_width;
 
-    TextWrap m_currentTextWrap { TextWrap::Wrap };
-    TextWrap m_lastObjectTextWrap { TextWrap::Wrap };
+    TextWrapMode m_currentTextWrap { TextWrapMode::Wrap };
+    TextWrapMode m_lastObjectTextWrap { TextWrapMode::Wrap };
```

Comment:
Maybe these could just be booleans instead of using the enumeration?

Reply:
It is possible, but I kind of like the explicitness of the enum values. Our style guide encourages this especially when passing arguments

Reply:
Yes but we don't pass these as arguments. It's OK if you want to leave this but the enumeration suggestion for the style guide is to avoid mysterious `false` and `true` arguments and constants, which is not a thing here. I pushed for that style guideline and I think it's sometimes misunderstood.